.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     - checkout
     - run:
```

Comment:
Why making changes to this file in this PR. This PR should only change the readme.